Magento_Catalog/web/js/quantity-update.babel.js

```diff
@@ -8,13 +8,15 @@ class QuantityUpdate {
   }
 
   events() {
-    this.plus.addEventListener('click', () => {
+    this.plus.addEventListener('click', e => {
       this.increment();
       this.evaluateConditions();
+      this.triggerInput(e);
     });
     this.minus.addEventListener('click', () => {
```

Comment:
I think here also should be `e` instead of `()`

Reply:
good catch, thanks!